web/instances/instances.go

```diff
@@ -85,6 +84,17 @@ func createHandler(c echo.Context) error {
 			return wrapError(err)
 		}
 	}
+	if iterations := c.QueryParam("KdfIterations"); iterations != "" {
+		iter, err := strconv.ParseInt(iterations, 10, 64)
+		if err != nil {
+			return wrapError(err)
+		}
+		if iter < 5000 && iter != 0 {
```

Comment:
Avec un nombre d'itérations recommandées de 100k, 5k semble vraiment très peu.

Reply:
On pourrait d'ailleurs extraire cette valeur minimum dans une constante. Dans le package `crypto` avec `DefaultPBKDF2Iterations` peut-être ?

Reply:
Pour les 5k, ça vient des implémentations en Go et Rust.